pyopenms_viz/_core.py

```
@@ -1181,322 +127,14 @@
-        f"Could not find plotting backend '{backend}'. Needs to be one of 'bokeh', 'matplotlib', or 'plotly'."
-    )
-
-
-def _get_plot_backend(backend: str | None = None):
-
-    backend_str: str = backend or "matplotlib"
-
-    if backend_str in _backends:
-        return _backends[backend_str]
+        # Use `marginal_z` for plotting the marginal distributions
+        plot_marginal_histogram(marginal_z)
```

Ruff (0.8.2)

Undefined name `plot_marginal_histogram`

(F821)